### PR TITLE
Added documentation howto

### PR DESCRIPTION
**Pull request by 3manuek**
Signed-off-by: 3manuek <emanuel@ongres.com>

### Description
Added documentation on the repository structure.
 
### Issues Resolved

[152](https://github.com/babelfish-for-postgresql/babelfish_project_website/issues/152)

### Check List
- [ ] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by susanmdouglas-aws — APPROVED** [review]

**Review by nasbyj — APPROVED** [review]

File: DOC_STRUCTURE.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Documentation structure of the Babelfish Website
+
+## Documentation Index and folder structure
+
+The list of the documentation collection is configured in the `_config.yml` file, in the `just_the_docs` attribute. This is the original configuration:
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:5]: How about:

The website headings are configured in the `config.yml` file, in the `just_the_docs` attribute. Each value provided in a `name` attribute will be an expandable heading in the left navigation panel:

File: DOC_STRUCTURE.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      nav_fold: true
+    internals:
+      name: Babelfish internals
+      nav_fold: true
+    faq:
+      name: FAQ and getting help
+      nav_fold: true
+```
+
+Each one of the `collections` attributes points to the homonimous folder name, prefixed with an underscore:
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:30]: How about:

Each name in a `collections` attributes points to the homonimous folder name, prefixed with an underscore:

File: DOC_STRUCTURE.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+```
+internals -> _internals/
+client -> _client/
+```
+
+Each folder contains the documentation for each section, and they'll appear as subelements in the documentation sidebar. Each markdown file must contain the following header format:
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:37]: How about:

Each folder contains the documents that reside in that section; each document will appear as link in the navigation sidebar. Each markdown file must contain the following header format:

File: DOC_STRUCTURE.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+```markdown
+---
+layout: default
+title: Compiling Babelfish from source
+nav_order: 2
+---
+```
+
+The header is self-explanatory, and you need to configure at least `title` and `nav_order` (which should be sequential for each section).
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:48]: How about:

The header is self-explanatory; you need to provide at least the `title` and `nav_order` (which should be sequential for each section).

File: DOC_STRUCTURE.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+Images should be placed in the `assets/images` folder. To use the image, you can add the following block to the markdown file:
+
+```html
+<img src="bg.png" data-src="img1.jpg" />
+```
+
+## Publishing posts
+
+Posts are different from documentation, as they reside in a different folder. The `_posts` folder contains the published posts in markdown format. The header of the posts should follow the below format:
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:60]: How about:

Posts are different from Babelfish documentation.  You can access posts from the menu in the upper-right corner of the Babelfish website landing page. Content for the posts section of the website resides in the `_posts` folder in markdown format. The header information for each post should follow the below format:

File: DOC_STRUCTURE.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+authors: 
+    - The Babelfish Project Team
+comments: true
+title: "Announcing Open Source Babelfish for PostgreSQL: An Accelerator for SQL Server Migration"
+categories:
+- releases
+---
+```
+
+> It is possible to commit a post and keep it hidden by using the `draft: true` attribute, which allows to work on a file without publishing it until draft is set to `false`.
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:75]: How about:

To commit a post, and keep it hidden, use the `draft: true` attribute.  This allows you to work on a file without publishing it.  To publish the file, set to ``draft: false`.

File: DOC_STRUCTURE.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+## Issue templates
+
+The issue templates are located in the `.github/ISSUE_TEMPLATE`, in yaml files.
+For more information about issue template, check [Github's documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository).
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:80]: How about:

For more information about issue templates, check [Github's documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository).

File: DOC_STRUCTURE.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+
+## Changing site index
+
+The site index resides in the `index.markdown` file, which contains the index sections.
+
+In this file, resides the sidebar and button callouts. The format is a yaml structure, placed in the markdown header.
```
Comment (susanmdouglas-aws) [inline DOC_STRUCTURE.md:87]: How about:

The site index resides in the `index.markdown` file.  The file defines the sidebar and button callouts and onscreen content. The format is a yaml structure, placed in the markdown header.

File: DOC_STRUCTURE.md (hunk 1 above)
Comment (3manuek) [reply DOC_STRUCTURE.md:5]: Just a note: `_config.yaml` isn't a typo, keeping this.

**Review by susanmdouglas-aws — APPROVED** [review]

**Review by gopinathpai — APPROVED** [review]

**Pull request merged**